### PR TITLE
#542: Add tests to ensure that the bad matches no longer match

### PR DESCRIPTION
**Pull request by jkondrat**
Organization emails and phones were replaced with fake ones

### CODE REVIEW

File: src/main/java/org/benetech/servicenet/converter/CSVFileConverter.java [hunk 1]
```diff
@@ -20,10 +21,9 @@ public CSVFileConverter(String delimiter) {
         }
     }
 
-    @Override
-    public String convert(MultipartFile file) throws IOException {
+    public String convert(InputStream is) throws IOException {
```
Comment (pmuchowski) [inline src/main/java/org/benetech/servicenet/converter/CSVFileConverter.java]: I think it should be private

File: src/test/java/org/benetech/servicenet/matching/MatchingAccuracyIntTest.java [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+            DocumentUpload upload = new DocumentUpload();
+            upload.setFilename(fileName);
+            uploads.add(upload);
+            data.add(loadCsv(SMC_DIR + fileName + CSV));
+        }
+        MultipleImportData importData = new MultipleImportData(data, uploads, new DataImportReport(), SMC_CONNECT_PROVIDER, true, null);
+        smcConnectDataAdapter.importData(importData);
+    }
+
+    private String loadCsv(String fileName) throws IOException {
```
Comment (pmuchowski) [inline src/test/java/org/benetech/servicenet/matching/MatchingAccuracyIntTest.java]: maybe you could move it to the utils class

File: src/main/java/org/benetech/servicenet/converter/CSVFileConverter.java (hunk 1 above)
Comment (jkondrat) [reply src/main/java/org/benetech/servicenet/converter/CSVFileConverter.java]: it's used in integration test

File: src/test/java/org/benetech/servicenet/matching/MatchingAccuracyIntTest.java (hunk 2 above)
Comment (jkondrat) [reply src/test/java/org/benetech/servicenet/matching/MatchingAccuracyIntTest.java]: done

**Pull request merged**